scripts/tombstone_bundles.py

```diff
@@ -43,10 +50,11 @@ def tombstone_bundle(uuid, version):
     else:
         raise Exception(f"Unable to verity tombstone {uuid}")
 
-bundles_to_tombstone = [
-    # Place bundles list here
-]
+bundles_to_tombstone = []
 
 for fqid in bundles_to_tombstone:
     uuid, version = fqid.split(".", 1)
     tombstone_bundle(uuid, version)
```

Comment:
Why not catch the exception here and dump the bundle uuid to stdout immediately?

That way no global variable is needed, and the list of failed bundles is not lost if the execution is aborted.

Reply:
```Retrying: RequestHistory(method='DELETE', url='/v1/bundles/fe9644c0-8815-41d2-9f58-c463e7240dc5?replica=aws&version=2019-05-02T232753.057217Z', error=None, status=500, redirect_location=None)
Retrying: RequestHistory(method='DELETE', url='/v1/bundles/fe9644c0-8815-41d2-9f58-c463e7240dc5?replica=aws&version=2019-05-02T232753.057217Z', error=None, status=500, redirect_location=None)
unable to tombstone: fe9644c0-8815-41d2-9f58-c463e7240dc5.2019-05-02T232753.057217Z too many 500 error responses : HTTPSConnectionPool(host='dss.dev.data.humancellatlas.org', port=443): Max retries exceeded with url: /v1/bundles/fe9644c0-8815-41d2-9f58-c463e7240dc5?replica=aws&version=2019-05-02T232753.057217Z (Caused by ResponseError('too many 500 error responses',))
tombstoning fe873053-ac7a-44af-9bea-3cc36f331ea3 2019-05-03T185717.488840Z
Retrying: RequestHistory(method='DELETE', url='/v1/bundles/fe873053-ac7a-44af-9bea-3cc36f331ea3?replica=aws&version=2019-05-03T185717.488840Z', error=None, status=500, redirect_location=None)
Retrying: RequestHistory(method='DELETE', url='/v1/bundles/fe873053-ac7a-44af-9bea-3cc36f331ea3?replica=aws&version=2019-05-03T185717.488840Z', error=None, status=500, redirect_location=None)
Retrying: RequestHistory(method='DELETE', url='/v1/bundles/fe873053-ac7a-44af-9bea-3cc36f331ea3?replica=aws&version=2019-05-03T185717.488840Z', error=None, status=500, redirect_location=None)
Retrying: RequestHistory(method='DELETE', url='/v1/bundles/fe873053-ac7a-44af-9bea-3cc36f331ea3?replica=aws&version=2019-05-03T185717.488840Z', error=None, status=500, redirect_location=None)```


@xbrianh  The operator would get both; they would be able to see the error inline during the execution, as well as after the script is finished so they don't have to search and compose a list for themselves.